System/Console/Haskeline/Backend/Posix.hsc

```diff
@@ -61,7 +63,7 @@ ehOut = eH . hOut
 -------------------
 -- Window size
 
-foreign import ccall ioctl :: FD -> CULong -> Ptr a -> IO CInt
+foreign import capi "sys/ioctl.h ioctl" ioctl :: FD -> CULong -> Ptr a -> IO CInt
```

Comment:
I'm wondering whether there's an edge case OS that this would break on, if ioctl is provided by another header.

But given the severity of the breakage (and my lack of access to any of them for testing) I think it's fine for us to merge/release this change now.